docs/plugins/panels.md

```diff
@@ -58,6 +65,28 @@ kind: "ScatterChart"
 spec: # TODO document the spec of ScatterChart
 ```
 
+## Table
+
+```yaml
+kind: "Table"
+spec:
+  [ density: <enum = "compact" | "standard" | "comfortable"> ]
+  [ columnSettings: <Column Settings specification> ]
+```
+
+### Column Settings specification
+
+```yaml
+name:  <string>
+[ header:  <string> ]
+[ headerDescription:  <string> ]
+[ cellDescription: <string> ]
+[ align: <enum = "left" | "center" | "right"> ]
+[ enableSorting: <boolean> ]
+[ width: <number | "auto"> ]
```

Comment:
not sure for the syntax here...

Reply:
I think we can just remove the square brackets

Reply:
Sorry I should have precised, I meant the syntax for the disjunction here, I think the "enum" we usually use (like for `align` right above) doesnt apply here because in the case of `width` the user can either provide a number or the "auto" string. Any other string is invalid.
In both CUE & typescript the syntax is as simple as `width?: number | 'auto';`